lib/command/init.js

```diff
@@ -21,7 +21,7 @@ const defaultConfig = {
   mocha: {},
 };
 
-const helpers = ['WebDriver', 'Puppeteer', 'TestCafe', 'Protractor', 'Nightmare', 'Appium', 'Playwright'];
+const helpers = require('../plugin/standardActingHelpers').slice().filter(e => e !== 'Mochawesome');
```

Comment:
Mochawesome is not **acting** helper. Why it is in this list?

Maybe we could have differnet lists of helpers and pick the right one (or merge differnet lists) depending on context?

Reply:
@DavertMik I'm not really sure why it exists in the `standardActingHelpers` list, would we remove it?